ctez.mligo

```diff
@@ -27,7 +27,7 @@ type parameter =
   | Mint_or_burn of mint_or_burn
   | Cfmm_price of nat * nat
   | Set_addresses of set_addresses
-  | Get_target of nat contract
+  | Get_target of (nat * nat) contract
```

Comment:
cfmm_price is a nat * nat but target is a nat, a bit arbitrary but I would rather minimally change ctez itself

Reply:
Agreed. I've just changed all instances of `target` to `nat`, instead of `nat * nat`